lib/strategies/webapp-strategy.js

```diff
@@ -262,8 +262,18 @@ function handleAuthorization(req, options = {}, strategy) {
 	redirectForAuthorization(req, options, strategy);
 }
 
-function isAuthenticated(req) {
-	return req.session[WebAppStrategy.AUTH_CONTEXT];
+function isAuthenticated(req, options = {}) {
+	if (!req.session[WebAppStrategy.AUTH_CONTEXT]) {
+		return false;
+	}
+	
+	const allowExpiredTokensOnSession = options.hasOwnProperty("allowExpiredTokensOnSession") ?
```

Comment:
lets do the opposite. (simply change the true to false)
make it validate by default and add not document this flag.
when publish SDK we will increase the Minor version.

Reply:
good idea